manifests/cf-manifest/deployments/030-cf-jobs.yml

```diff
@@ -1,6 +1,9 @@
 meta:
   environment: ~
 
+  api_domain: (( concat "api." properties.domain ))
+  apps_domain: (( concat "apps." properties.domain ))
```

Comment:
Do we need to purchase and configure two separate wildcards certificates for the router?